### DISCUSSION TITLE
add viewing a users posted images to their profile

### DISCUSSION THREAD

**Discussion by gouwsbrendon** [civitai/civitai#262] [Ideas]
Quite self explanitory. Would be really cool to showcase users posted content. I wanted to credit someones post elswhere and got this idea

**Comment by Machinekng** [comment]
User image galleries is an in-dev feature:

https://www.reddit.com/r/civitai/comments/10t0xdu/heres_how_were_planning_on_improving_browsability/

**Reply by xDarkAgentx** [reply]
This would be awesome to have. A Personal gallery attached to your profile. Nothing too complicated.
